airbyte-cdk/bulk/core/load/build.gradle

```diff
@@ -10,3 +10,29 @@ dependencies {
     testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.8.1")
     implementation "org.jetbrains.kotlin:kotlin-reflect:2.0.20"
 }
+
+project.sourceSets {
```

Comment:
mostly just copied this from https://github.com/airbytehq/airbyte/pull/45886, but trimmed down to just the stuff we actually need. in particular - `src/integrationTest/*` is automatically added as a source set if you just... declare the `integrationTest` source set :shrug:

(everything else seems  necessary though)